src/main/java/org/opensrp/service/EventService.java

```diff
@@ -35,27 +36,70 @@
 
 	public static final String GROWTH_MONITORING_EVENT = "Growth Monitoring";
 
+	public static final String GROWTH_MONITORING_EVENT_UNDERSCORED = "growth_monitoring";
+
 	public static final String VACCINATION_EVENT = "Vaccination";
 
 	public static final String OUT_OF_AREA_SERVICE = "Out of Area Service";
 
 	public static final String NEW_OUT_OF_AREA_SERVICE = "out_of_area_service";
 
+	public static final String RECURRING_SERVICE_UNDERSCORED = "recurring_service";
+
+	public static final String RECURRING_SERVICE = "Recurring Service";
+
 	public static final String NFC_CARD_IDENTIFIER = "NFC_Card_Identifier";
 
 	public static final String CARD_ID_PREFIX = "c_";
 
 	private static final String OPENSRP_ID = "opensrp_id";
 
+	public static final String RECURRING_SERVICE_TYPES = "recurring_service_types";
+
+	public static final String NUMERIC = "numeric";
+
+	public static final String DOSE = "dose";
+
+	public static final String DATE = "date";
+
+	private static final String CODED = "coded";
+
+	public static final String _DOSE = "_dose";
+
+	public static final String _DATE = "_date";
+
+	public static final String CONCEPT = "concept";
+
 	private final EventsRepository allEvents;
 
-	private ClientService clientService;
+	private final ClientService clientService;
+
+	private final TaskGenerator taskGenerator;
+
+	private final PlanRepository planRepository;
+
+	private final ExportEventDataMapper exportEventDataMapper;
+
+	private final SimpleDateFormat simpleDateFormat = new SimpleDateFormat("yyyy-MM-dd");
+
+	public interface ObsConstants {
+
+		String CODED_FIELD_VALUE = "1065AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA";
+		String NUMERIC_FIELD_CODE = "1639AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA";
+		String ITN_CODED_FIELD_CODE = "159855AAAAAAAAAAAAAAAAAAAAAAAAAAAAAA";
+		String ITN_DATE_FIELD_CODE = "159432AAAAAAAAAAAAAAAAAAAAAAAAAAAAAA";
+		String VIT_A_CODED_FIELD_CODE = "161534AAAAAAAAAAAAAAAAAAAAAAAAAAAAAA";
+		String DEWORMING_CODED_FIELD_CODE = "159922AAAAAAAAAAAAAAAAAAAAAAAAAAAAAA";
+	}
+
+	interface RecurringServiceConstants {
 
-	private TaskGenerator taskGenerator;
+		String DEWORMING = "deworming";
 
-	private PlanRepository planRepository;
+		String VIT_A = "vit_a";
 
-	private ExportEventDataMapper exportEventDataMapper;
+		String ITN = "itn";
+	}
```

Comment:
Does it make sense to move all these constants to a constants file?

Reply:
Moved.